### PR TITLE
Fix missing path for wishlists in wishlists/index

### LINKED ISSUE #78: Fix missing path for wishlists in wishlists/index

### PR DESCRIPTION
**Pull request by shadre**
Closes #78

### CODE REVIEW

**Comment by agalkiewicz** [comment]
:+1:

File: app/views/wishlists/index.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <ul>
     <% @wishlists.each do |wishlist| %>
       <li>
-        <%= link_to wishlist.name, "#" %>
+        <%= link_to wishlist.name, wishlist_path(wishlist) %>
```
Comment (student-tomasz) [inline app/views/wishlists/index.html.erb]: `wishlist` without even the helper call would work as well.

**Comment by student-tomasz** [comment]
👍

**Comment by Kacper3331** [comment]
:+1:

**Comment by gotar** [comment]
you can merge

**Pull request merged**